src/Transport/Receiving/MessagePump.cs

```diff
@@ -96,6 +96,11 @@ public async Task StartReceive(CancellationToken cancellationToken = default)
                 AutoCompleteMessages = false
             };
 
+            if (transportSettings.MaxAutoLockRenewalDuration.HasValue)
+            {
+                receiveOptions.MaxAutoLockRenewalDuration = transportSettings.MaxAutoLockRenewalDuration.Value;
```

Comment:
Agreed we should follow the SDK's default, but we should document this as it might result in behavior changes when upgrading to new versions of the SDK.

Reply:
Documented it in the corresponding internal design document